airflow/providers/sftp/triggers/sftp.py

```diff
@@ -134,7 +134,7 @@ async def run(self) -> AsyncIterator[TriggerEvent]:
                 # Break loop to avoid infinite retries on terminal failure
                 break
 
-        yield TriggerEvent({"status": "error", "message": exc})
+        yield TriggerEvent({"status": "error", "message": str(exc)})
```

Comment:
@Taragolis This PR also had to fix the code bugs though it doesn't come under PT012 fix directly. Is it ok to commit those code changes in this PR or do I have to raise a separate PR with code fix followed by changes closely related to PT012

Reply:
I think we should separate that one.

Reply:
Actually - it will be fine if we change tile and make it a fix to that one - that also fixes Pt102 - tha will be actually better.